x-pack/plugins/fleet/server/services/agents/actions.ts

```diff
@@ -31,6 +32,8 @@ const ONE_MONTH_IN_MS = 2592000000;
 
 export const NO_EXPIRATION = 'NONE';
 
+const SIGNED_ACTIONS: Set<Partial<AgentActionType>> = new Set(['UNENROLL', 'UPGRADE']);
+
 export async function createAgentAction(
```

Comment:
@joeypoon We should probably sign actions in the `bulkCreateAgentActions` method too no?